packages/@atjson/source-gdocs-paste/src/converter.ts

```diff
@@ -141,21 +168,18 @@ GDocsSource.defineConverterTo(OffsetSource, doc => {
       );
       let lastEnd = start;
       for (let paragraphBoundary of paragraphBoundaries) {
-        doc.addAnnotations(
-          new ParseAnnotation({
-            start: paragraphBoundary.start,
-            end: paragraphBoundary.end,
-            attributes: {
-              reason: "new line paragraph separator"
-            }
-          })
-        );
-
         if (lastEnd < paragraphBoundary.start) {
           doc.addAnnotations(
             new Paragraph({
               start: lastEnd,
               end: paragraphBoundary.start
+            }),
```

Comment:
Oh, this is where we are changing the "wrap all list items in paragraphs" behaviour. The follow up for this is https://github.com/CondeNast/atjson/issues/406

Reply:
This does not wrap all list items in paragraphs. We only do so if the list item strictly contains a paragraph separator.